.vscode/launch.json

```diff
@@ -7,6 +7,18 @@
             "name": "wgpu_room(debug)",
             "program": "${workspaceFolder}/examples/target/debug/wgpu_room",
             "preLaunchTask": "build"
+        },
```

Comment:
I don't have a strong opinion, but do you think we should commit this launch configuration?